extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/RedisClientProcessor.java

```diff
@@ -39,4 +72,101 @@ HealthBuildItem addHealthCheck(RedisBuildTimeConfig buildTimeConfig) {
     RuntimeInitializedClassBuildItem initializeBulkTypeDuringRuntime() {
         return new RuntimeInitializedClassBuildItem(BulkType.class.getName());
     }
+
+    @BuildStep
+    @Record(ExecutionTime.RUNTIME_INIT)
+    public void produceRedisClient(RedisClientRecorder recorder, ApplicationArchivesBuildItem applicationArchives,
+            BuildProducer<SyntheticBeanBuildItem> syntheticBeans,
+            VertxBuildItem vertxBuildItem) {
+        Set<String> clientNames = new HashSet<>();
+        clientNames.add(RedisClientUtil.DEFAULT_CLIENT);
+
+        IndexView indexView = applicationArchives.getRootArchive().getIndex();
+        Collection<AnnotationInstance> clientAnnotations = indexView.getAnnotations(REDIS_CLIENT_ANNOTATION);
+        for (AnnotationInstance annotation : clientAnnotations) {
+            clientNames.add(annotation.value().asString());
+        }
+
+        for (String clientName : clientNames) {
+            syntheticBeans.produce(createBlockingSyntheticBean(recorder, clientName));
+            syntheticBeans.produce(createReactiveSyntheticBean(recorder, clientName));
+            syntheticBeans.produce(createMutinyRedisApiSyntheticBean(recorder, clientName));
+            syntheticBeans.produce(createMutinyRedisClientSyntheticBean(recorder, clientName));
+            syntheticBeans.produce(createVertxRedisAPISyntheticBean(recorder, clientName));
```

Comment:
The one two lines above is called `Api` not `API`. Could we be consistent?

Reply:
Good point, thanks for spotting this. I had this mixed up all over the codebase and I've applied some clean up.